TeachingPlatform/subjects/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+
+class Subject(models.Model):
+    name = models.CharField(max_length=128)
+    comments = models.TextField()
```

Comment:
Shouldn't it be called `comment`? If you want to keep many comments here, create a relation instead :slightly_smiling_face: